search/grammar_variables_matcher.go

```diff
@@ -134,7 +134,7 @@ func programPositionWithoutTermMatch(vMap map[string][]string, cm cache.CacheMan
 		return false
 	}
 	if varProgramCollection == "" {
-		return true
+		varProgramCollection = consts.PROGRAM_COLLECTION_NEW_LIFE
```

Comment:
Why not return, what is changing in this line?

Reply:
When we index intents of just a chapter number without specific collection, this overrides landing page intents of specific year.
For example when searching for '2019', instead of finding conventions landing page of that year, it tries to bring program with chapter number 2019 (which is not exist).
By setting "varProgramCollection = consts.PROGRAM_COLLECTION_NEW_LIFE" we limit the maximum value of 'chapter' variable to the last program of 'new life'.